packages/create-llama/helpers/index.ts

```diff
@@ -120,28 +121,43 @@ const installDependencies = async (
   }
 };
 
-const copyContextData = async (root: string, contextFile?: string) => {
+const copyContextData = async (
+  root: string,
+  dataSource?: TemplateDataSource,
+) => {
   const destPath = path.join(root, "data");
-  if (contextFile) {
-    console.log(`\nCopying provided file to ${cyan(destPath)}\n`);
-    await fs.mkdir(destPath, { recursive: true });
-    await fs.copyFile(
-      contextFile,
-      path.join(destPath, path.basename(contextFile)),
-    );
-  } else {
-    const srcPath = path.join(
-      __dirname,
-      "..",
-      "templates",
-      "components",
-      "data",
-    );
-    console.log(`\nCopying test data to ${cyan(destPath)}\n`);
-    await copy("**", destPath, {
-      parents: true,
-      cwd: srcPath,
-    });
+
+  let dataSourceConfig = dataSource?.config as FileSourceConfig;
+
+  // Copy file
+  if (dataSource?.type === "file") {
+    let srcFile =
+      dataSourceConfig.path ??
+      path.join(__dirname, "..", "templates", "components", "data", "101.pdf");
```

Comment:
can we just use the whole `components/data` folder like before? if we rename the pdf file this code will break

Reply:
because we provide an option as `An example PDF` so it should copy a file rather than a whole folder. If we change the file name, then the test will fail.

Reply:
Can't we treat the option `An example PDF` so that we're copying the `components/data` folder? Then whatever is in this folder will be used